activerecord/lib/active_record/type/decimal.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module ActiveRecord
+  module Type
+    class Decimal < ActiveModel::Type::Decimal # :nodoc:
+      POSITIVE_INFINITY = BigDecimal("Infinity").freeze
```

Comment:
Why the `freeze` here?

Reply:
`BigDecimal`s aren't frozen by default and since I'm using it as a constant, I froze it.